src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -157,45 +157,42 @@ public class SendReceiveTest : ClientWebSocketTestBase
                 var recvBuffer = new byte[100];
                 var recvSegment = new ArraySegment<byte>(recvBuffer);
 
-                try
+                for (int i = 0; i < tasks.Length; i++)
                 {
-                    for (int i = 0; i < tasks.Length; i++)
-                    {
-                        tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
-                    }
+                    tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
+                }
 
-                    Task.WaitAll(tasks);
+                Exception e = Record.Exception(() => Task.WaitAll(tasks));
```

Comment:
> Note, also, that Assert.Throws can't be used here, since there is no guarantee that an exception will actually be thrown

That's true here (I was basing my comment on the title of this PR, saying that you were replacing Assert.True(false)s, and didn't look to see whether this case had one).  It's not in the previous case I highlighted, where we should use Assert.ThrowsAnyAsync.

And here since Assert.True(false) isn't being used, I'm not seeing what the transformation to use Record.ExceptionAsync really helps with.